tesseract/integration-test/src/lib.rs

```diff
@@ -392,57 +318,51 @@ async fn get_request_works() -> Result<(), anyhow::Error> {
 	let _ = log_setup();
 	let (chain_a_sub_client, chain_b_sub_client) = create_clients().await?;
 
-	log::info!(" \n 🧊integration test for para: 2000 to para 2001: get request \n");
+	log::info!("🧊integration test for para: 2000 to para 2001: get request \n");
 
 	// =======================================================================
 	let (chain_a_client, chain_b_client) =
 		(Arc::new(chain_a_sub_client.clone()), Arc::new(chain_b_sub_client.clone()));
 
-	let (client_a, client_b) =
-		(chain_a_sub_client.clone().client, chain_b_sub_client.clone().client);
-
-	// Accounts & keys
-	let dave_signer = PairSigner::<Hyperbridge, _>::new(
-		Pair::from_string("//Dave", None).expect("Unable to create Dave account"),
-	);
 	// parachain info pallet fetching para id
 	let encoded_chain_b_id_storage_key =
 		"0x0d715f2646c8f85767b5d2764bb2782604a74d81251e398fd8a0a4d55023bb3f";
+	let key = hex::decode(encoded_chain_b_id_storage_key.strip_prefix("0x").unwrap()).unwrap();
 
 	let latest_height_b =
-		chain_a_client.query_latest_height(chain_b_client.state_machine_id()).await? - 5;
+		chain_a_client.query_latest_height(chain_b_client.state_machine_id()).await? - 10;
```

Comment:
Use a finalized height, and it should be the exact height in the state machine update event.